packages/cadl-ranch-specs/http/azure/resource-manager/models/common-types/managed-identity/main.tsp

```diff
@@ -0,0 +1,162 @@
+model ManagedIdentityTrackedResourceProperties {
+  @visibility("read")
+  @doc("The status of the last operation.")
+  provisioningState: string;
+}
+
+@armResourceOperations
+interface ManagedIdentityTrackedResources {
+  @scenario
+  @scenarioDoc("""
+  Resource GET operation.
+  Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+  Expected query parameter: api-version=2023-12-01-preview
+
+  Expected response body:
+  ```json
+  {
+    "id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+    "location": "eastus",
+    "tags": {
+      "tagKey1": "tagValue1"
+    },
+    "identity": {
+      "type": "SystemAssigned",
+      "principalId": <any uuid string>
+      "tenantId": <any uuid string>
+     },
+    "properties": {
+      "provisioningState": "Succeeded"
+    }
+  }
+  ```
+  """)
+  get is ArmResourceRead<ManagedIdentityTrackedResource>;
+
+  @scenario
+  @scenarioDoc("""
+  Resource PUT operation.
+  Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+  Expected query parameter: api-version=2023-12-01-preview
+  Expected request body:
+  ```json
+  {
+    "location": "eastus",
+    "tags": {
+      "tagKey1": "tagValue1"
+    },
+    "properties": {},
+    "identity": {
+      "type": "SystemAssigned"
+    }
+  }
+  ```
+   Expected response body:
+  ```json
+  {
+    "id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+    "location": "eastus",
+    "tags": {
+      "tagKey1": "tagValue1"
+    },
+    "identity": {
+      "type": "SystemAssigned",
+      "principalId": <any uuid string>,
+      "tenantId": <any uuid string>
+     },
+    "properties": {
+      "provisioningState": "Succeeded"
+    }
+  }
+  ```
+  """)
+  createWithSystemAssigned is ArmResourceCreateOrReplaceSync<ManagedIdentityTrackedResource>;
+
+  @scenario
+  @scenarioDoc("""
+  Resource PATCH operation.
+  Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.Models.CommonTypes.ManagedIdentity/managedIdentityTrackedResources/identity",
+  Expected query parameter: api-version=2023-12-01-preview
+  Expected request body:
+  ```json
+  {
+    "identity": {
+      "type": "SystemAssigned,UserAssigned",
+      "userAssignedIdentities": {
+        "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/id1": {}
```

Comment:
Could you try in any real service, whether non-empty value is allowed?

e.g.
```
"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/id1": {
  "principalId": <any uuid string>,
  "clientId": <any uuid string>
}
```
in request

Reply:
1. The member variables in `ManagedServiceIdentityProperty` are all read-only, so the value cannot be set when creating and updating Identity.
2. In addition, the key of `userAssignedIdentities` is the id of `managed Identity`, so I think the `principalId` and `clientId` of each usesigned managed Identity will be filled by the backend according to the id of managed Identity.
So your idea may not be realized.

Reply:
User can directly use the `ManagedServiceIdentityProperty` from a previous response. They don't need to set the value. The response is de-serialized by the SDK and it contains these values.

Reply:
OK, I checked the serialize code, it won't serialize this 2 properties. We are good.

```java
    public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
        jsonWriter.writeStartObject();
        return jsonWriter.writeEndObject();
    }
```